### PR TITLE
Add DASH ACL Group

### PR DESCRIPTION
**Pull request by marian-pritsak**
Provide ability do attach/detach ACL
dynamically via a group

Signed-off-by: Marian Pritsak <marianp@mellanox.com>

### CODE REVIEW

File: experimental/saiexperimentaldash.h [hunk 1]
```diff
@@ -205,6 +205,72 @@ typedef enum _sai_eni_attr_t
...
+    /**
+     * @brief Action set_eni_attrs parameter INBOUND_STAGE3_DASH_ACL_GROUP_ID
+     *
+     * @type sai_object_id_t
+     * @flags CREATE_AND_SET
+     * @objects SAI_OBJECT_TYPE_DASH_ACL_GROUP
+     * @allownull true
+     * @default SAI_NULL_OBJECT_ID
+     */
+    SAI_ENI_ATTR_INBOUND_STAGE3_DASH_ACL_GROUP_ID,
```
Comment (prsunny) [inline experimental/saiexperimentaldash.h:239]: Can we add this till stage5.

File: experimental/saiexperimentaldash.h [hunk 2]
```diff
@@ -205,6 +205,72 @@ typedef enum _sai_eni_attr_t
...
+    /**
+     * @brief Action set_eni_attrs parameter INBOUND_STAGE1_DASH_ACL_GROUP_ID
+     *
+     * @type sai_object_id_t
+     * @flags CREATE_AND_SET
+     * @objects SAI_OBJECT_TYPE_DASH_ACL_GROUP
+     * @allownull true
+     * @default SAI_NULL_OBJECT_ID
+     */
+    SAI_ENI_ATTR_INBOUND_STAGE1_DASH_ACL_GROUP_ID,
```
Comment (prsunny) [inline experimental/saiexperimentaldash.h:217]: We can have STAGE1_ACL_V4 and V6 separate

File: experimental/saiexperimentaldash.h (hunk 1 above)
Comment (marian-pritsak) [reply experimental/saiexperimentaldash.h]: Done

File: experimental/saiexperimentaldash.h (hunk 2 above)
Comment (marian-pritsak) [reply experimental/saiexperimentaldash.h]: Done

**Comment by marian-pritsak** [comment]
DASH_ACL is renamed to DASH_ACL_RULE (requested by the community)

**Review by kcudnik — APPROVED** [review]

**Review by prsunny — APPROVED** [review]

**Pull request merged**